README.md

```diff
@@ -3,3 +3,20 @@
 [![CircleCI](https://circleci.com/gh/meeshkan/worlds-greatest-website.svg?style=svg)](https://circleci.com/gh/Meeshkan/worlds-greatest-website)
 
 Thanks for checking out the [world's greatest website](http://meeshkan.com). We use [11ty](https://github.com/11ty/eleventy). Pull requests welcome!
+
+## Development
+
+Install dependencies:
+
+```bash
+$ npm i
+```
+
+Run build:
+
+```bash
+$ npm run build
+```
+
+See other available scripts in [package.json](./package.json).
```

Comment:
Would we maybe want to mention the `npm run serve` command on the README? I find it's the nicest for development but maybe that's just me 🤷‍